### ISSUE TITLE
Exception thrown in preload script doesn't stop electron

### ISSUE THREAD

**Issue by zaggino** [atom/electron#1566]
Sample init code:
```
win = new BrowserWindow({
        width: 800,
        height: 600,
        preload: require.resolve("./preload")
    });
```
Sample preload script would be:
```
throw new Error("YOU SHALL NOT PASS");
```
Nothing about this is shown in the console (no log message) and electron continues execution with loading app url like nothing happened.

Using `"electron-prebuilt": "0.25.2"`

**Comment by paulcbetts** [comment]
We're hitting this as well, #1551 is my proposed fix

**Comment by zcbenz** [comment]
Fixed by #1551.
